tests/test_fitter_error_checking.py

```diff
@@ -134,6 +134,22 @@ def test_jump_everything_wideband():
         assert not np.isnan(fitter.model[p].value)
 
 
+def test_null_vector():
+    model = get_model(io.StringIO("\n".join([par_base])))
+    model.free_params = ["ELONG", "ELAT"]
+    toas = make_fake_toas(58000, 58900, 10, model, obs="barycenter", freq=1400.0, dm=15.0, dm_error=1e-4*u.pc*u.cm**-3)
+    fitter = pint.fitter.GLSFitter(toas, model)
+    with pytest.warns(pint.fitter.DegeneracyWarning, match=r".*degeneracy.*following parameter.*ELONG\b"):
+        fitter.fit_toas(threshold=1e-14)
+    for p in fitter.model.free_params:
+        assert not np.isnan(fitter.model[p].value)
+    fitter = pint.fitter.WidebandTOAFitter(toas, model)
```

Comment:
Could you put the GLSFitter and the WidebandTOAFitter in separate tests? That way if the tests fail we'll know which one, or if it's both.

Reply:
You could actually parameterize this test over WidebandTOAFitter, GLSFitter, and also WLSFitter with `@pytest.mark.parametrize()` to have one piece of code test all three.